vk_video_decoder/test/vulkan-video-dec/CMakeLists.txt

```diff
@@ -0,0 +1,127 @@
+set(VULKAN_VIDEO_DEC_SOURCES
```

Comment:
Can we use directly the library from libs/vkVideoDecoder or shall include all these files in my project ?
Can this file be simplified to use the library directly as for vkVideoEncoder ?

Reply:
create a simple test which is embedded a minimum of files to demonstrate the decode of a file with some options, same of the test for the encode library?